internal/cli/resume.go

```diff
@@ -0,0 +1,231 @@
+	}
+
+	return nil
+}
+
+func (o *ResumeOptions) Run(ctx context.Context, args []string) error {
+	c, err := o.BuildClient()
+	if err != nil {
+		return fmt.Errorf("creating client: %w", err)
+	}
+
+	_, name, err := parseAndValidateKindName(args[0])
+	if err != nil {
+		return err
+	}
+
+	// Handle bulk resume case (name is empty for plural forms)
+	if name == "" {
+		return o.runBulkResume(ctx, c)
+	}
+
+	// Handle single device case (name is provided)
+	return o.runSingleResume(ctx, c, name)
+}
+
+func (o *ResumeOptions) runSingleResume(ctx context.Context, c *apiclient.ClientWithResponses, name string) error {
+	// Use the bulk resume API with a field selector to target the specific device
+	fieldSelector := fmt.Sprintf("metadata.name=%s", name)
+	request := api.DeviceResumeRequest{
+		FieldSelector: &fieldSelector,
+	}
+
+	response, err := c.ResumeDevicesWithResponse(ctx, request)
+	if err != nil {
+		return fmt.Errorf("resuming device %s: %w", name, err)
+	}
+
+	if response.HTTPResponse != nil {
+		switch response.HTTPResponse.StatusCode {
+		case http.StatusOK:
+			fmt.Printf("Resume request for %s \"%s\" completed\n", DeviceKind, name)
+		case http.StatusBadRequest:
+			return fmt.Errorf("invalid request for device %s", name)
+		default:
+			return fmt.Errorf("unsuccessful resume request for device %s: %s", name, string(response.Body))
+		}
+	}
+
+	return nil
+}
+
+func (o *ResumeOptions) runBulkResume(ctx context.Context, c *apiclient.ClientWithResponses) error {
+	request := api.DeviceResumeRequest{}
+
+	// Build selector description for error messages
+	var selectorDesc string
+
+	if o.All {
+		selectorDesc = "all devices"
+		// When --all is specified, use a field selector that matches all devices
+		allDevicesSelector := "metadata.name!=''"
+		request.FieldSelector = &allDevicesSelector
+	} else {
+		if o.LabelSelector != "" {
+			request.LabelSelector = &o.LabelSelector
+		}
+		if o.FieldSelector != "" {
+			request.FieldSelector = &o.FieldSelector
+		}
+
+		if o.LabelSelector != "" && o.FieldSelector != "" {
+			selectorDesc = fmt.Sprintf("label selector '%s' and field selector '%s'", o.LabelSelector, o.FieldSelector)
+		} else if o.LabelSelector != "" {
+			selectorDesc = fmt.Sprintf("label selector '%s'", o.LabelSelector)
+		} else {
+			selectorDesc = fmt.Sprintf("field selector '%s'", o.FieldSelector)
+		}
+	}
+
+	response, err := c.ResumeDevicesWithResponse(ctx, request)
+	if err != nil {
+		return fmt.Errorf("resuming devices with %s: %w", selectorDesc, err)
+	}
+
+	if response.HTTPResponse != nil {
+		switch response.HTTPResponse.StatusCode {
+		case http.StatusOK:
+			if response.JSON200 != nil {
+				count := response.JSON200.ResumedDevices
+				fmt.Printf("Resume operation completed:\n")
+				fmt.Printf("  Devices resumed: %d\n", count)
+
+				if count == 0 {
+					fmt.Printf("No devices matched the selector or were in conflictPaused state\n")
```

Comment:
Should be `ConflictPaused`.
And this new status is missing in the user documentation. 
https://github.com/flightctl/flightctl/blob/main/docs/user/device-api-statuses.md?plain=1#L9